tests/acceptance/django_settings.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+SECRET_KEY = "test"
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
+        "NAME": os.environ.get("PGDATABASE_DJANGO", "procrastinate"),
+        "USER": os.environ.get("PGUSER", "postgres"),
+        "PASSWORD": os.environ.get("PGPASSWORD", "password"),
+        "HOST": os.environ.get("PGHOST", "localhost"),
```

Comment:
As far as I can tell, only the NAME is mandatory. Other variables will already default to the libpq env vars if not present. We could keep ENGINE and NAME.

Also, is there a specific reason for using `PGDATABASE_DJANGO` instead of `PGDATABASE` and could we make it default to `PGDATABASE` if `PGDATABASE_DJANGO` is not provided ?